pages/index.js

```diff
@@ -156,7 +157,8 @@ export default function Home() {
     }
   }
 
-  function handleContactClick(id) {
+  async function handleContactClick(id) {
+    const photoURL = await Storage.get(currentContact.id);
```

Comment:
This throws an error for me, because currentContact hasn't been loaded yet, so we can't reference it. currentContact gets set below.